### PR TITLE
No entrypoint, cmd, or command

### PR DESCRIPTION
**Pull request by baude**
When an image does not have an ENTRYPOINT nor a CMD and the
user does not provide a command in the CLI, we should fail
gracefully.

This resolves issue #328

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: test/e2e/run_entrypoint_test.go [hunk 1]
```diff
@@ -28,6 +28,17 @@ var _ = Describe("Podman run entrypoint", func() {
...
+	It("podman run no command, entrypoint, or cmd", func() {
+		dockerfile := `FROM docker.io/library/alpine:latest
+ENTRYPOINT []
+CMD []
+`
+		podmanTest.BuildImage(dockerfile, "foobar.com/entrypoint:latest")
+		session := podmanTest.Podman([]string{"run", "foobar.com/entrypoint:latest"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(125))
+	})
```
Comment (TomSweeneyRedHat) [inline test/e2e/run_entrypoint_test.go]: consider adding a test here for the number of containers, s/b zero.  Don't hold the submit for that if you decide to do it.

**Comment by TomSweeneyRedHat** [comment]
LGTM and all happy green buttons.

**Comment by mheon** [comment]
@rh-atomic-bot r+
